sfall/Arrays.cpp

```diff
@@ -250,6 +250,7 @@ void LoadArrays(HANDLE h) {
 		if (nextarrayid == 0) nextarrayid++;
 		arrays.insert(array_pair(nextarrayid, arrayVar));
 		savedArrays[arrayVar.key] = nextarrayid++;
+		arrayVar.keyHash.clear();
```

Comment:
Not at all. What this does is it clears *keyHash* of a temporary *sArrayVar* instance used as a buffer during the loading operation **after** its contents have been copied to the *array_pair* that's inserted to *arrays*. If you don't clear the buffer *keyHash*, you are going to have stale elements inside it when loading following arrays. That's what this commit fixes.